alignak/objects/satellitelink.py

```diff
@@ -241,23 +240,24 @@ def add_failed_check_attempt(self, reason=''):
         if self.attempt == self.max_check_attempts:
             self.set_dead()
 
-    def update_infos(self):
+    def update_infos(self, now):
         """Update satellite info each self.check_interval seconds
         so we smooth arbiter actions for just useful actions.
         Create update Brok
 
         :return: None
         """
         # First look if it's not too early to ping
-        now = time.time()
-        since_last_check = now - self.last_check
-        if since_last_check < self.check_interval:
+        if (now - self.last_check) < self.check_interval:
             return
 
         self.last_check = now
 
         # We ping and update the managed list
         self.ping()
+        if not self.alive or self.attempt > 0:
```

Comment:
Not need to run the end of the function (check conf managed) if satellite not available because we will try connection again for nothing because 0.2 second before it says not available